php/context/class-fieldmanager-context.php

```diff
@@ -100,4 +100,33 @@ protected function render_field( $args = array() ) {
 		}
 	}
 
+	/**
+	 * Register a field for use with the REST API.
+	 *
+	 * @since 1.3.0
+	 *
+	 * @param string|array $object_type Required. The object type in the REST API where this field will be available.
+	 */
+	public function register_rest_field( $object_type ) {
+		// Ensure the REST API is active and the field wants to be shown in REST.
+		if ( $this->fm->can_show_in_rest() ) {
+			register_rest_field(
+				$object_type,
+				$this->fm->name,
+				array(
+					'get_callback'    => array( $this, 'rest_get_callback' ),
+					'update_callback' => array( $this, 'rest_update_callback' ),
```

Comment:
These callbacks are no longer part of this class.

Reply:
Correct, but since they are declared within `Fieldmanager_Context_Storable`  it works fine.